docs/entities/ListingOffer.md

```diff
@@ -0,0 +1,30 @@
+# ListingOffer
```

Comment:
It is preferable to avoid complex words when naming main entities. You can read about aggregate roots from domain driven design, but you want to name entities as an expert in this domain would call it. For example, in the context of a marketplace like ours I thought of this at the beginning as a Listing, but maybe it is more adequate if this is a Product.